controller/ExcludedOS.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+declare(strict_types=1);
+
+namespace oat\taoClientDiagnostic\controller;
+
+use oat\taoClientDiagnostic\model\exclusionList\ExcludedOSClassService;
+
+/**
+ *
+ * @package oat\taoClientDiagnostic\controller
+ */
+class ExcludedOS extends \tao_actions_SaSModule
+{
+    public function editInstance()
+    {
+        $clazz = $this->getCurrentClass();
+        $instance = $this->getCurrentInstance();
+        $myFormContainer = new \tao_actions_form_Instance($clazz, $instance);
```

Comment:
Just a small suggestion, I'd avoid using variables to store values that you'll use as arguments for another method only.

```
$myFormContainer = new \tao_actions_form_Instance($this->getCurrentClass(), $this->getCurrentInstance());
```

Reply:
Good catch, thank you.
However, I think I cannot apply the suggestion fully, since the `$instance` variable is used elsewhere. But I will at least do it for the class.